conf/routes

```diff
@@ -98,4 +98,5 @@ POST /direct-debit/check-account                      controllers.DirectDebit.ch
 # ----- Assets ----- #
 
 GET  /assets/*file                                  controllers.Assets.at(path="/public/compiled-assets", file)
+GET  /assets-payment/*file                          controllers.Assets.at(path="/public/assets-new-ui", file)
```

Comment:
@Ap0c @regiskuckaertz you're probably already aware but just wanted to point out that this is one bit of cleanup that has to happen before the test goes out. Since this bypasses the asset hashing mechanism we'd have no way of busting browser caches for these assets and if we needed to deploy any fixes to the test, users could end up with different assets out of sync with each other.

Quite possibly this will automatically be cleaned up by the work to move the HTML into React and the CSS into Sass - but don't want to forget about it

Reply:
Yes, I've moved all the CSS in React-land in my two latest PRs 👍 This route can be removed once I no longer need the temporary static page.